src/org/rascalmpl/library/util/Maven.java

```diff
@@ -78,15 +79,27 @@ private static void setField(CliRequest req, String fieldName, Object value) thr
         field.set(req, value);
     }
     
-    private static CliRequest buildRequest(String[] args, ISourceLocation manifestRoot) throws ReflectiveOperationException {
+    private static CliRequest buildRequest(String[] args, @Nullable ISourceLocation manifestRoot) throws ReflectiveOperationException {
         // we need to set a field that the default class doesn't set
         // it's a work around around a bug in the MavenCli code
         var cons = CliRequest.class.getDeclaredConstructor(String[].class, ClassWorld.class);
         cons.setAccessible(true);
         var result = cons.newInstance(args, null);
-        var manifestRootFile = new File(manifestRoot.getPath());
-        setField(result, "workingDirectory", manifestRootFile.getPath());
-        setField(result, "multiModuleProjectDirectory", manifestRootFile);
+
+        if (manifestRoot != null) {
+            var manifestRootFile = new File(manifestRoot.getPath());
+            setField(result, "workingDirectory", manifestRootFile.getPath());
+            setField(result, "multiModuleProjectDirectory", manifestRootFile);
+        }
+        else {
+            try {
+                setField(result, "multiModuleProjectDirectory", File.createTempFile("dummy", ""));
+            }
+            catch (ReflectiveOperationException | IOException e) {
+                // ignore
```

Comment:
Can we explain int he comment why we ignore this one, while the other `setFields` are allowed to throw?